Yoast/javascript
# Add colors and icons used in plugin

## Summary


This PR can be summarized in the following changelog entry:

* Adds colors and icons used in the plugin

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* This can be tested with https://github.com/Yoast/wordpress-seo/pull/14885
* See test instructions on that PR

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
